data/games/guilty_gear_xrd_rev_2/movespecs/answer.json

```diff
@@ -137,41 +159,47 @@
           "j.214K",
           "j.214S",
           "j.214H"
-        ]
+        ],
+        "imageId": "savvy_ninpo_request_for_approval"
       },
       {
         "name": "Cancel Stance",
         "inputs": [
           "22"
         ],
+        "imageId": "savvy_ninpo_seal_of_approval",
```

Comment:
This is a nitpick but as you noted in the image index, this is the image for hanging from a scroll, not canceling off of one. I think I'd prefer we not display an image here rather than use a wrong (though related) one. What do you think?